bricks/r13n/hooks/packages/arb_parser/pubspec.yaml

```diff
@@ -3,9 +3,9 @@ description: A simple `arb` file parser.
 publish_to: none
 
 environment:
-  sdk: ">=2.19.0 <3.0.0"
```

Comment:
We can use 2.19 if we set r13n to be 2.19 compatible. Do we already have an issue to track that migration plan for this package?

Reply:
I believe the e2e test fails if this uses 2.19.0. Since the e2e runs on the minimum supported SDK and the latest SDK version. This package is a dependency the brick's hook so running on the minimum supported SDK fails when using the minimum supported SDK.

I've opened another issue to bump to 2.19.0 [here](https://github.com/VeryGoodOpenSource/r13n/issues/47).